### ISSUE TITLE
sync with different tdrop command submit

### ISSUE THREAD

**Issue by K1ngst0m** [noctuid/tdrop#368]
for example, I decided to use the two tdrop commands to pop up the terminal and hide (with shortcuts), and the command like this:

``` bash
tdrop -am -w 2300 -h 1150 -x 1500 -y 1000 kitty --class kitty_tdrop
```

but my CPU now is busy so the first tdrop command is not done when executing the second tdrop command, tdrop will pop up twice so there exist two terminals now.

is there some existing solution I can use to sync two commands even out of tdrop?

**Comment by noctuid** [comment]
You want a command to open and then immediately close a window?  Or am I completely misunderstanding?  Could you rephrase the behavior you want to have happen so that I can understand better?

If you want to just have something happen immediately after tdrop is done creating a window, you could do `tdrop <command 1> && tdrop <command2>`, but I assume you want something more complex than that?

**Comment by K1ngst0m** [comment]
sorry, I forgot to check this issue, and I found a workaround with wm built-in methods.
